tests/__init__.py

```diff
@@ -2727,6 +2727,21 @@ def test_streaming_subscription(self):
         finally:
             self.account.inbox.unsubscribe_from_notifications(subscription_id)
 
+    def test_multifolder_streaming_subscription(self):
+        folders = []
+        for folder in self.account.root.walk():
+            folders.append(folder)
+
+        subscription_id = self.account.subscribe_for_notifications(folders, CONCRETE_EVENT_CLASSES)
+        try:
+            for event in self.account.listen_for_notifications(subscription_id, timeout_s=60):
```

Comment:
Is there a way to verify that the events came from more than a single folder?

Reply:
Not really. These tests are talking to a real Exchange server which means we don't have control over which of notifications come in. Eventually it would be nice to mock out the HTTP communication though.

Reply:
Boo, alright